spec/system/task_spec.rb

```diff
@@ -85,8 +80,7 @@
 
       it '既にステータスが完了のタスクのステータスを変更した場合、Taskの完了日が更新されないこと' do
         # TODO: FactoryBotのtraitを利用してください
-        project = FactoryBot.create(:project)
-        task = FactoryBot.create(:task, project_id: project.id, status: :done, completion_date: Time.current.yesterday)
+        task = FactoryBot.create(:task, :project_date)
```

Comment:
let!を使うことで変数の定義でコード量を増やすことを回避できます。
定義する場所はここと同じとは限らないので、どの場所が適切かも考えて見てください。
```suggestion
        let!(:task) { create(:task, :project_date) }
```

Reply:
a87b2ceでlet!で定義し直し定義名、記述箇所を修正しました。

Reply:
a87b2ce でlet!で定義し直し定義名、記述箇所を修正しました。